docker-build-package

```diff
@@ -116,7 +116,12 @@ elif [ "${SAVE_ORIG}" == "true" ]; then
 fi
 mkdir -p "${output_dir}"
 
-[[ "${SAVE_ORIG}" != "true" ]] && cp -v output/opensource/orig/*orig.tar.* "${output_dir}/"
+# we build source packages only for amd64 and for non-commercial
+# (non source closed) components. there is no need to copy the orig
+# file if we do not need it, since later, during publication to the apt-repo
+# we clean them from incoming based on changes file, which refers to orig files
+# only for source builds. See MEN-5879.
+[[ "${SAVE_ORIG}" != "true" && "$commercial" != "true" && "$ARCH" == "amd64" ]] && cp -v output/opensource/orig/*orig.tar.* "${output_dir}/"
```

Comment:
@lluiscampos  this one does (I think) exactly what you said, it gets orig files only when we build source packages.

Reply:
Not _exactly_ , because it still copies all orig files (say, mender-artifact, mender-connect, etc) for every single build. My approach was more strict by being explicit on taking only the required orig file. And, on top of it, I recommended using `mv` and not `cp` to have a clean build and avoid the `exclude:` key that we have now in the pipeline job.

Does it make sense?

Reply:
but the exclude is so nice, it is obvious that the build package job should not publish the orig directory. and that was your comment:
> build:packages:<distro>:<version>:amd64 produces /output/opensource/<distro>-<version>-amd64/*.orig.tar.gz but does not propagate output/opensource/orig/*.orig.tar.gz.

Reply:
> the exclude is so nice,

Sorry, but to me looks like a workaround, not a "nice" design.

> it is obvious that the build package job should not publish the orig directory. and that was your comment: 
> > build:packages:::amd64 produces /output/opensource/--amd64/_.orig.tar.gz but does not propagate output/opensource/orig/_.orig.tar.gz.

It was a comment from high level. I didn't mean achieving that by excluding them (in CI), but designing it so that the files are not passed along (in the script).

Reply:
I removed the exclude. but I strongly disagree: exclude is something that is built for this reason.

Reply:
@lluiscampos it does not work without the exclude, isnt gitlab caring over them automatically and the exclude is needed?

Reply:
effect: all no source builds have orig files carried over. I can explicitly remove them, but come on, this is what we have exclude for.

Reply:
Discussed in a call :telephone_receiver: